test/rollbax/client_test.exs

```diff
@@ -23,8 +23,7 @@ defmodule Rollbax.ClientTest do
     assert body =~ ~s("environment":"test")
     assert body =~ ~s("level":"warn")
     assert body =~ ~s("body":"pass")
-    assert body =~ ~s("foo":"bar")
-    assert body =~ ~s("qux":"custom")
+    assert Poison.decode!(body)["data"]["custom"] == %{"foo" => "bar", "qux" => "custom"}
```

Comment:
Let's keep matching with `=~` here instead of decoding. I know it didn't catch the bug in the first place and this is bad, but it's better to update all tests at once to use the parsed body IMO.